gr-zeromq/lib/base_impl.cc

```diff
@@ -143,6 +148,11 @@ base_source_impl::base_source_impl(int type,
 #endif
     }
 
+    // Set ZMQ_LINGER to 0 so socket won't infinitely block during teardown
+    // Re: Github #1132
+    const int linger_default = 1000;
+    d_socket.setsockopt(ZMQ_LINGER, &linger_default, sizeof(linger_default));
```

Comment:
Put the 1000 in one place and reuse, to not repeat.

Reply:
Do you have a suggestion on the "right" place to initialize the const so that it is in the correct scope? That was what I was unsure of, so I figured having it on the line above worked well enough.

Reply:
Probably safest to put it in this same file, in an anonymous namespace as a global variable. Do use `constexpr`, though.

```
namespace {
constexpr int LINGER_DEFAULT = 1000; // 1 second.
}
```

Nonmember const should be capitalized per styleguide.